src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/HaplotypeCallerArgumentCollection.java

```diff
@@ -81,12 +70,7 @@
     @Advanced
     @Argument(fullName = "indelSizeToEliminateInRefModel", shortName = "ERCIS", doc = "The size of an indel to check for in the reference model", optional = true)
     public int indelSizeToEliminateInRefModel = 10;
-    
-    /**
-     * Bases with a quality below this threshold will reduced too the minimum usable qualiy score (6).
-     */
-    @Argument(fullName = "base_quality_score_threshold", shortName = "bqst", doc = "Base qualities below this threshold will be reduced to the minimum (" + QualityUtils.MIN_USABLE_Q_SCORE + ")", optional = true)
-    public byte BASE_QUALITY_SCORE_THRESHOLD = PairHMM.BASE_QUALITY_SCORE_THRESHOLD;
```

Comment:
I think `pcrErrorMode` belongs in `LikelihoodEngineArgumentCollection.java` too; it's a parameter for the likelihood engine. Then `createLikelihoodCalculationEngine` doesn't need to take `pcrErrorMode` as an argument (but this is not the primary reason for moving it. We want to move it because it semantically belongs in `LikelihoodEngineArgumentCollection.java`)